### DISCUSSION TITLE
Help to create multi sheet excel file

### DISCUSSION THREAD

**Discussion by thiagoolsilva** [jxlsteam/jxls#275] [Q&A] [answered]
Hi, I'd love to use the feature named  multi sheet, but I've some problems to understand how to split specific  data inserted in the context to create all sheets with its own content. 

I did some research without any success. Is it possible to someone create some example  covering this topic?

Thanks!

**Comment by gastendonk** [comment, marked as answer]
Every (multisheet-) sheet has the same design but other data. If every sheet must have its own design you must design all sheets in the template and delete them at report-creation-time using Apache POI code if needed.

Checkout jxls repository and look at MultiSheetMarkup1Demo and similar files.

https://github.com/search?q=repo%3Ajxlsteam%2Fjxls%20multisheet&type=code

**Comment by thiagoolsilva** [comment]
Thanks! I've solved my problem.
